src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <section class="calendar">
+      <div class="calendar__day calendar__day--first__day-sat"></div>
```

Comment:
```suggestion
    <section class="calendar calendar--first__day-sat">
      <div class="calendar__day"></div>
```
first day should be modifier of calendar block
and what about of implementing month-length modifier?

Reply:
![image](https://user-images.githubusercontent.com/48822777/235366291-805a700c-aaf6-421f-9f96-e1f2d8c5ce08.png)